meta/src/utils.rs

```diff
@@ -0,0 +1,32 @@
+use proc_macro2::TokenStream as TokenStream2;
+use quote::quote;
+use syn::{Attribute, Field, Ident, Meta, PathSegment, Type};
+
+// https://users.rust-lang.org/t/how-to-use-a-vector-of-tokenstreams-created-with-quote-within-quote/81092
```

Comment:
```suggestion
/// Implements a block of functions for a struct.
/// 
/// Issue that presents how to map a vector of `TokenStream` within quote! macro:
/// https://users.rust-lang.org/t/how-to-use-a-vector-of-tokenstreams-created-with-quote-within-quote/81092
```

Reply:
Added doc comment, but moved hyperlink to ordinary code comment; I don't think implementation details belong in project documentation.